triples-generator/src/main/scala/ch/datascience/triplesgenerator/reprovisioning/TriplesVersionCreator.scala

```diff
@@ -44,19 +44,27 @@ private class IOTriplesVersionCreator(
   import ch.datascience.rdfstore.SparqlQuery
   import eu.timepit.refined.auto._
 
-  override def insertCliVersion(): IO[Unit] = updateWitNoResult {
+  override def updateCliVersion(): IO[Unit] = updateWitNoResult {
     val entityId = (renkuBaseUrl / "cli-version").showAs[RdfResource]
     SparqlQuery(
       name = "cli version create",
       Set(
         "PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>",
         "PREFIX renku: <https://swissdatasciencecenter.github.io/renku-ontology#>"
       ),
-      s"""|INSERT DATA { 
-          |  $entityId rdf:type renku:CliVersion.
-          |  $entityId renku:version '$currentCliVersion'.
-          |}
-          |""".stripMargin
+      s"""DELETE {$entityId renku:version ?o}
```

Comment:
I think if the line looks like this `s"""|DELETE {$entityId renku:version ?o}` scalafmt could format the whole query a bit better.